internal/cmdinit/cmdinit_test.go

```diff
@@ -51,24 +52,40 @@ packageMetadata:
 
 	b, err = ioutil.ReadFile(filepath.Join(d, "my-pkg", man.ManFilename))
 	assert.NoError(t, err)
-	assert.Equal(t, `my-pkg
-==================================================
+	assert.Equal(t, strings.ReplaceAll(`# my-pkg
 
-# NAME
+## Description
+my description
 
-  my-pkg
+## Usage
 
-# SYNOPSIS
+### Fetch the package
+'kpt pkg get REPO_URI[.git]/PKG_PATH[@VERSION] my-pkg'
+Details: https://googlecontainertools.github.io/kpt/reference/pkg/get/
 
-  kubectl apply --recursive -f my-pkg
+### Update package from upstream
```

Comment:
I suggest showing a __workflow__ to immediately get started with a package (Instead of a reference guide)

I this case, If I've just fetched a package, it doesn't make to run update on it immediately.

Eventually we should have a diagram of a workflow we can render inline. It's worth a thousand words.

Reply:
Removed the update step from the flow. I think it now covers the most likely step needed to get started with a package.